cohortextractor/presto_utils.py

```diff
@@ -17,13 +18,22 @@ def presto_connection_from_url(url):
     if "PFX_PATH" in os.environ:
         adapt_connection(conn, conn_params)
 
+    if "providerplus.emishealthinsights.co.uk" in url:
+        certs_dir = (
+            Path(__file__).resolve().parent.parent
+            / "certs"
+            / "providerplus.emishealthinsights.co.uk"
+        )
+        session.verify = certs_dir / "2.crt"
+
     return ConnectionProxy(conn)
 
 
 def adapt_connection(conn, conn_params):
     """Adapt connection to use passphrase-protected PKCS#12 certificate.
 
-    See instructions in opensafely/accessing-emis-data/ for getting a certificate.
+    For instructions for getting a certificate, see
+    https://ebmdatalab.github.io/datalab-team-manual/opensafely/accessing-emis-data/
```

Comment:
💖

Reply:
Why this change, OOI? I think we want OS related docs in the OS org, private if needed?

Reply:
Oh, is this just an explicit url?

Reply:
Just an explicit URL.